doc/sphinx/addendum/type-classes.rst

```diff
@@ -320,6 +320,10 @@ Summary of the commands
    :attr:`universes(polymorphic)`, :attr:`universes(template)`,
    :attr:`universes(cumulative)` and :attr:`private(matching)` attributes.
 
+   .. remark::
+      Not to be confused with the notion of "class" from
+      `implicit coercions<classes-implicit-coercions>`.
+
```

Comment:
For more clarity/less confusion, I would look at rewording the chapter to use "typeclass" everywhere instead of "class".  Are you up for it?

It still makes sense to have the note, but as a parenthetical comment right after the definition of "typeclass".  (It still a good idea to distinguish "typeclass" from "class".  If people routinely say "class" when they mean "typeclass", that's worth mentioning in the doc.

Reply:
That idea is good in theory, though I worry about the vernac syntax `Class` which means typeclass.

Reply:
You could add a parenthetical comment in the description of the `Class` command.  The first sentence of that description already mentions typeclasses.

BTW, I see `Print TypeClasses` uses camel case while elsewhere it's `typeclasses` or `Typeclasses`.  You think that's worth changing?

Reply:
> BTW, I see `Print TypeClasses` uses camel case while elsewhere it's `typeclasses` or `Typeclasses`. You think that's worth changing?

Yes